pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>org.ohdsi</groupId>
     <artifactId>authenticator</artifactId>
-    <version>0.0.2-SNAPSHOT</version>
+    <version>0.0.3-SNAPSHOT</version>
```

Comment:
The PR is not valid since it introduces breaking changes but only patch version is raised. Overall, I believe that the required functionality could be implemented w/o the breaking changes so that all dependant projects wouldn't be affected

Reply:
fixed